ga4gh/_protocol_definitions.py

```diff
@@ -746,6 +863,84 @@ def __init__(self, **kwargs):
         """
 
 
+class FeatureGroup(ProtocolElement):
```

Comment:
Yes, FeatureGroup is somewhat confusing.

Sean Upchurch <notifications@github.com> writes:
> In ga4gh/_protocol_definitions.py:
> 
> > @@ -746,6 +863,84 @@ def __init__(self, **kwargs):
> >          """
> >
> >
> > +class FeatureGroup(ProtocolElement):
> 
> This is part of RNA and represents an associated group of quantifications - for
> example a collection of transcripts from the same gene. If the name is too
> confusing due to overload of the word 'feature' we can change it in future.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub*
>